Rules/CommonScripts/ScoreboardRender.as

```diff
@@ -371,7 +371,11 @@ float drawScoreboard(CPlayer@[] players, Vec2f topleft, CTeam@ team, Vec2f emble
 					1 : 0),             5,     0,         0,
 				(acc.map_contributor ?
 					1 : 0),             6,     0,         0,
-				(acc.moderation_contributor && (!isSpecial(p) || coloredNameEnabled(getRules(), p)) ? //ensure accolade is visible for past admins
+				(acc.moderation_contributor && (
```

Comment:
won't nitpick you to change this right now. But in the future for readability it would be better to split this out into multiple booleans then combine them in the final check.